docker/README.md

```diff
@@ -0,0 +1,144 @@
+# dGen Docker Usage Guide
+
+By default, the dgen container uses the [Deleware residential dataset](https://oedi-data-lake.s3.amazonaws.com/dgen/de_final_db/dgen_db.sql).
+
+You can customize the dataset, see the [Customizing the Dataset](#customizing-the-dataset) section below.
+
+### Mac and Linux quick start
+
+This quickstart uses docker-compose to run dgen. The default path to store dgen data files and excel configurations is ~/dgen_data/.  This path is shared with your running containers, you can change this path but you will need to edit the `docker-compose.yml` to reflect the data directory of your choice.
+
+Prerequisites assume you are using a Mac and you already installed [Docker Desktop](https://docs.docker.com/desktop/setup/install/mac-install/)
+
+##### Create the data directory
+```bash
+$ mkdir -p ~/dgen_data/
+$ chmod 755 ~/dgen_data/
+$ ls -l ~/dgen_data/ # Its expected to be empty, after starting dgen you will see data files in this location.
+```
+
+##### Startup the dgen containers
+``` bash
+$ cd dgen/docker/
+$ docker-compose up --build -d
+[+] Running 2/2
+ ✔ Container dgen_1     Started 0.1s
+ ✔ Container postgis_1  Started 0.0s
+```
+
+##### Connect to the running containers
+``` bash
+$ docker attach dgen_1 # Attach to dgen environment container
```

Comment:
```suggestion
$ docker attach dgen_1 # Attach to dGen environment container
```
Would it be better to remove the word "environment" from the comment to avoid any confusion with the Conda environment? Fine with whichever you think is better!

Reply:
I would agree, I can see where it would be confusing.  I am going to call it the dGen container.